LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentLauncher.kt

```diff
@@ -62,12 +65,12 @@ class LocalPaymentLauncher internal constructor(private val browserSwitchClient:
      * completing the authentication flow.
      */
     fun handleReturnToAppFromBrowser(
-        pendingRequest: LocalPaymentPendingRequest.Started,
-        intent: Intent
+            pendingRequest: LocalPaymentPendingRequest.Started,
+            intent: Intent
```

Comment:
Not sure if the changes in this file were auto formatted but it looks super far in